admin/bin/qserv-data-loader.py

```diff
@@ -52,9 +52,9 @@
 
 """
 
-#--------------------------------
+# -------------------------------
```

Comment:
My editor (pycharm, which is very good and light IDE) tell me it's not PEP8 compliant.